.changeset/bright-cougars-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+Add loading state on search input
```

Comment:
I think we should start to do something like this for our changelogs to they're easer to visually scan/parse (look at [last release](https://github.com/hashicorp/design-system/releases/tag/%40hashicorp%2Fdesign-system-components%402.7.1) to see what I mean)

```suggestion
`Form::TextInput` - Add loading state on "search" type
```